CHANGES.md

```diff
@@ -1,5 +1,27 @@
 ## unreleased
 
+## v2.5.0
+
+### New features
+* add context reinitialisation (PR #490)
+* add Dolmen frontend (PR #491,#541,#545)
+* add model generation (PR #530)
```

Comment:
This is misleading, model generation was already present. What we did was modernize the support for model generation, and add compatibility with the smtlib2 interface.

I think that #530 and #614 should be bundled together here (and also #659 #703 #614 #609 #755 if we want to be complete)